### PR TITLE
Add $wgMinervaTalkAtTop to isvwiki

### PR DESCRIPTION
**Pull request by scrapter**
Per T7732

### CODE REVIEW

**Review by RhinosF1 — CHANGES_REQUESTED** [review]
Indentation

**Comment by scrapter** [comment]
> Indentation

Remove it?

**Comment by scrapter** [comment]
@RhinosF1 I think I fixed it.

File: LocalSettings.php [hunk 1]
```diff
@@ -3224,6 +3224,18 @@
 	'wgMinervaAlwaysShowLanguageButton' => [
 		'default' => true,
 	],
+        'wgMinervaTalkAtTop' => [
```
Comment (Universal-Omega) [inline LocalSettings.php:3227]: Indentation

**Review by Universal-Omega — CHANGES_REQUESTED** [review]

File: LocalSettings.php (hunk 1 above)
Comment (scrapter) [reply LocalSettings.php:3227]: Okay, I’m confused, do I need to add more indentation, or remove it?

Comment (Universal-Omega) [reply LocalSettings.php:3227]: Well, you have alot of spaces this should be tabbed not spaces.

Comment (scrapter) [reply LocalSettings.php:3227]: Ohhh, I get it now. Alright, I’ll put tabs.

Comment (scrapter) [reply LocalSettings.php]: Done.

Comment (Universal-Omega) [reply LocalSettings.php]: Indentation is still messed up, it doesn't use spaces anymore but still incorrectly indented.

Comment (scrapter) [reply LocalSettings.php]: How about now?

Comment (Universal-Omega) [reply LocalSettings.php]: That's not it either.

Comment (Universal-Omega) [reply LocalSettings.php]: Still not it.

Comment (scrapter) [reply LocalSettings.php]: Sorry if I’m wasting your time.

Comment (scrapter) [reply LocalSettings.php]: Is it good now?

Comment (RhinosF1) [reply LocalSettings.php]: No

**Comment by Reception123** [comment]
Done in #4034. Please do have a look at that PR so you can do better for the next PR.

**Pull request closed without merging**